lang/scala/scala.talon

```diff
@@ -24,15 +24,19 @@ settings():
     user.code_protected_variable_formatter = "PRIVATE_CAMEL_CASE"
     user.code_public_variable_formatter = "PRIVATE_CAMEL_CASE"
 
-state package: "package "
+state {user.scala_modifier}:
+    insert(user.scala_modifier)
+    key("space")
 
-state val: "val "
-state var: "var "
+state {user.scala_keyword}:
+    insert("{scala_keyword}")
```

Comment:
It's weird that modifiers get space added to them by the command while keywords include the space in them. It would be clearer if it was consistent.

Reply:
You are absolutely right. I have now included this in the commit.